docs/notebook_version_standardizer.py

```diff
@@ -89,7 +89,10 @@ def _standardize_outputs(notebooks):
 def _check_delete_empty_cell(notebook, delete=True):
     with open(notebook, "r") as f:
         source = json.load(f)
-        cell = source["cells"][-1]
+        try:
+            cell = source["cells"][-1]
+        except:
+            return True
```

Comment:
I'm going to back this change out before I merge.  For some reason when I run make lint-fix it fails here.